src/config/dynamic-theme-fixes.config

```diff
@@ -3328,6 +3328,20 @@ INVERT
 
 ================================
 
+kraftfuttermischwerk.de
+
+INVERT
+.title-container img.logo
```

Comment:
Doesn't look good. It's a image with a fixed background color, shouldn't be touched.